test/parallel/test-repl.js

```diff
@@ -836,8 +837,16 @@ function startUnixRepl() {
   ]);
 }
 
-function event(ee, eventName) {
-  return new Promise((resolve) => {
-    ee.once(eventName, common.mustCall(resolve));
+function event(ee, expected) {
+  return new Promise((resolve, reject) => {
+    const timeout = setTimeout(() => {
+      reject(
+        new Error(`Repl did not reply as expected for:\n\n${inspect(expected)}`)
+      );
```

Comment:
Is `expected` always an event name (a string?). 
If it is I think padding with two newlines is a bit much. It could just go right after the `: `.

Reply:
No, expected is the expected output, so either a string or an array with many strings.